index.html

```diff
@@ -0,0 +1,241 @@
+            <div class="fas fa-bars" id="menu-btn"></div>
+          </div>
+
+      </div>
+
+    </header>
+
+  </section>
+
+  <!-- HERO SECTION STARTS -->
+
+  <section class="hero">
+    <div class="hero-content">
+      <h5>Class Is Permanent</h5>
+      <h1 class="hero-title">We are the Porsche Club of America</h1>
+      <p class="hero-para">The Porsche Club of America is the largest single-marque car club in the world.</p>
+      <h3 class="hero-date"> 2022.11.24 (THU) ~ 28 (FRI)</h3>
+      <p class="hero-venue">
+        @ National Harbor, Maryland, USA
+      </p>
+    </div>
+  </section>
+
+  <!-- HERO SECTION ENDS -->
+
+  <!-- MAIN PROGRAM SECTION STARTS -->
+
+  <section class="main-program" id="program">
+    <h1 class="program-heading">Main Program</h1>
+    <div class="underline"></div>
+
+    <div class="program-cards">
+      
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Boxster</h3>
+        <p class="program-card-para">The 718 Boxster takes its name from a combination.</p>
+      </div>
+
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Cayenne</h3>
+        <p class="program-card-para">While the answer appear to be that the famous Porsche SUV.</p>
+      </div>
+
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Carrera</h3>
+        <p class="program-card-para">A model of the 911 in modern times, the Carrera.</p>
+      </div>
+
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Macan</h3>
+        <p class="program-card-para">The small SUV known as the “Macan”.</p>
+      </div>
+
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Panamera</h3>
+        <p class="program-card-para">The Porsche Panamera is, as far as anyone knows.</p>
+      </div>
+
+    </div> 
+    
+    <a href="#" class="cta-text">SEE THE ENTIRE CAR GARAGE</a>
+    <button type="button" class="ctb-btn">Join Our Grand Re-opening</button>
+
+  </section>
+
+  <!-- MAIN PROGRAM SECTION ENDS -->
+
+  <!-- FEATURED SPEAKERS SECTION STARTS -->
+
+  <section class="famous-people">
+    <div class="famous-people-top">
+      <h1 class="famous-people-title">Celebrities with Porsche</h1>
+      <div class="famous-people-underline"></div>
+    </div>
+
+    <div class="famous-people-loop-holder">
+
+    </div>
+    
+  </section>
+
+  <!-- FEATURED SPEAKERS SECTION ENDS -->
+
+  <!-- PARTNERS SECTION STARTS -->
+
+  <section class="partners" id="partners-desktop">
```

Comment:
[Optional]

- [ ] It's better if you set `object-fit: cover;` on the images of the partners section to remove the stretching 👍 

![image](https://user-images.githubusercontent.com/50721479/203852681-5d08dc73-3fac-46b0-b640-296e65be406b.png)